src/lib/RepositoryForms/View/ViewParametersListener.php

```diff
@@ -222,7 +222,7 @@ function (Repository $repository) use ($content) {
         }
 
         if (null === $location) {
-            throw new InvalidArgumentException('$location', 'Location for published content has to be provided');
+            throw new InvalidArgumentException('$location', 'you must provide a Location for the published Content item');
```

Comment:
```suggestion
            throw new InvalidArgumentException('$location', 'You must provide a Location for the published Content item');
``` 
I marked this because other messages start with a capital letter